eyeshade/controllers/owners.js

```diff
@@ -352,6 +352,8 @@ v1.unlinkPublisher = {
       }
       await publishers.update({ publisher: publisher }, state, { upsert: true })
 
+      await tokens.remove({ publisher: publisher }, { justOne: false })
```

Comment:
Is the `tokens` variable in scope here?

Reply:
commited the *wrong* file... sigh. fixed.

Reply:
sorry and many thanks!